drivers/SmartThings/matter-sensor/src/air-quality-sensor/init.lua

```diff
@@ -379,12 +379,12 @@ local function unit_conversion(value, from_unit, to_unit)
   local conversion_function = conversion_tables[from_unit][to_unit]
   if conversion_function == nil then
     log.info_with( {hub_logs = true} , string.format("Unsupported unit conversion from %s to %s", unit_strings[from_unit], unit_strings[to_unit]))
-    return 1
+    return
```

Comment:
I think it's probably better to return `nil`, "<error message"> and then you could check for nil and log the error message at one place in the calling function

Reply:
Good idea, this sounds cleaner - made these changes in the latest commit [0c5e8c0](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1582/commits/0c5e8c01430002ce682e0eb4253de9001369a5f7)